source/NVDAObjects/JAB/__init__.py

```diff
@@ -333,7 +340,12 @@ def _get_states(self):
 		return stateSet
 
 	def _get_value(self):
-		if self.role not in [controlTypes.Role.CHECKBOX,controlTypes.Role.MENU,controlTypes.Role.MENUITEM,controlTypes.Role.RADIOBUTTON,controlTypes.Role.BUTTON] and self._JABAccContextInfo.accessibleValue and not self._JABAccContextInfo.accessibleText:
+		if self.role not in [
+			controlTypes.Role.TOGGLEBUTTON, controlTypes.Role.CHECKBOX,
+			controlTypes.Role.MENU, controlTypes.Role.MENUITEM,
+			controlTypes.Role.RADIOBUTTON, controlTypes.Role.BUTTON] \
+		and self._JABAccContextInfo.accessibleValue \
+		and not self._JABAccContextInfo.accessibleText:

```

Comment:
Using parentheses improves readability and means you don't have to escape the newlines

```suggestion
		if (
			self.role not in [
				controlTypes.Role.TOGGLEBUTTON, controlTypes.Role.CHECKBOX,
				controlTypes.Role.MENU, controlTypes.Role.MENUITEM,
				controlTypes.Role.RADIOBUTTON, controlTypes.Role.BUTTON
			]
			and self._JABAccContextInfo.accessibleValue
			and not self._JABAccContextInfo.accessibleText
		):
```